sbx/trusted_services_list.yaml

```diff
@@ -21,3 +21,14 @@ clients:
     requireProofKey: false
     jwkSetUrl: "https://verifier.dome-marketplace-sbx.org/oidc/did/did:key:zDnaezLhSFWRZ1zrYQRwLxV8nDWmb2REHoUf7N3qvna1TFina"
     tokenEndpointAuthenticationSigningAlgorithm: "ES256"
+  - clientId: "did:key:zDnaehmkEhoybLgRkVbKpAv47Vu81Rw5TmLUA5PrRKuWSbhxn"
+    url: "https://dome-certification.dome-marketplace-sbx.org"
+    redirectUris: ["https://dome-certification.dome-marketplace-sbx.org/auth/vc/callback"]
+    scopes: ["openid_learcredential"]
+    clientAuthenticationMethods: ["client_secret_jwt"]
+    authorizationGrantTypes: ["authorization_code"]
+    postLogoutRedirectUris: ["https://dome-certification.dome-marketplace-sbx.org/"]
+    requireAuthorizationConsent: false
+    requireProofKey: false
+    jwkSetUrl: ""
```

Comment:
This parameter needs to be filled. You can use the verifier as a resolver at https://verifier.dome-marketplace-sbx.org/oidc/did/<your-did-key>. This way, the verifier can resolve it and obtain the jwks resolving the did. Otherwise, you would need to use the JWKS endpoint of your application where you are exposing the key.

Reply:
OK!!
So, from the did
you extract the coordinates and return them in that endpoint, in case I need to verify my signature.